tests/schema/test.firefox-schemas-import.js

```diff
@@ -82,6 +82,50 @@ describe('firefox schema import', () => {
         required: [],
       });
     });
+
+    it('handles an allOf with one being optional', () => {
```

Comment:
I think you should say "with one property being optional" or something like that because this isn't really clear.